frontend/file/src/main/java/com/virtuslab/gitmachete/frontend/file/highlighting/MacheteAnnotator.java

```diff
@@ -107,6 +105,24 @@ private void processMacheteGeneratedEntry(MacheteGeneratedEntry macheteEntry, An
     } catch (IOException e) {
       throw new RuntimeException(e);
     }
+
+    if (!branchNames.contains(processedBranchName)) {
+      try {
+        val parentBranchName = getParentBranchName(file, processedBranchName);
+        val basicAnnotationBuilder = holder
+            .newAnnotation(HighlightSeverity.ERROR,
+                getNonHtmlString("string.GitMachete.MacheteAnnotator.cannot-find-local-branch-in-repo")
+                    .format(processedBranchName))
+            .range(branch);
+        if (parentBranchName.isEmpty()) { // do not suggest creating a new root branch
+          basicAnnotationBuilder.create();
+        } else { // suggest creating a new branch from the parent branch
+          basicAnnotationBuilder.withFix(new CreateBranchQuickFix(processedBranchName, parentBranchName, file)).create();
+        }
+      } catch (BranchLayoutException e) {
+        throw new RuntimeException(e);
```

Comment:
Huh probably throwing an exception here is not a good choice:

```
In file: file:///home/plipski/git-machete-intellij-plugin/.git/machete

java.lang.RuntimeException: com.virtuslab.branchlayout.api.BranchLayoutException: Line no 9 in branch layout file (/home/plipski/git-machete-intellij-plugin/.git/machete) has unexpected indentation character inconsistent with previous one. Problematic line number: 9
	at com.virtuslab.gitmachete.frontend.file.highlighting.MacheteAnnotator.processMacheteGeneratedEntry(MacheteAnnotator.java:123)
	at com.virtuslab.gitmachete.frontend.file.highlighting.MacheteAnnotator.annotate(MacheteAnnotator.java:55)
	at com.intellij.codeInsight.daemon.impl.DefaultHighlightVisitor.runAnnotators(DefaultHighlightVisitor.java:131)
	at com.intellij.codeInsight.daemon.impl.DefaultHighlightVisitor.visit(DefaultHighlightVisitor.java:108)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.runVisitors(GeneralHighlightingPass.java:347)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.lambda$collectHighlights$7(GeneralHighlightingPass.java:279)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.analyzeByVisitors(GeneralHighlightingPass.java:306)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.lambda$analyzeByVisitors$8(GeneralHighlightingPass.java:309)
	at com.intellij.codeInsight.daemon.impl.DefaultHighlightVisitor.analyze(DefaultHighlightVisitor.java:93)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.analyzeByVisitors(GeneralHighlightingPass.java:309)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.collectHighlights(GeneralHighlightingPass.java:276)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.collectInformationWithProgress(GeneralHighlightingPass.java:221)
	at com.intellij.codeInsight.daemon.impl.ProgressableTextEditorHighlightingPass.doCollectInformation(ProgressableTextEditorHighlightingPass.java:97)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:57)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$1(PassExecutorService.java:400)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1154)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:393)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$12(CoreProgressManager.java:608)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:683)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:639)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:607)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:392)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$0(PassExecutorService.java:368)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:174)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:215)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:366)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:184)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: com.virtuslab.branchlayout.api.BranchLayoutException: Line no 9 in branch layout file (/home/plipski/git-machete-intellij-plugin/.git/machete) has unexpected indentation character inconsistent with previous one. Problematic line number: 9
	at com.virtuslab.branchlayout.impl.readwrite.BranchLayoutFileReader.parseToArrayRepresentation(BranchLayoutFileReader.java:7)
	at com.virtuslab.branchlayout.impl.readwrite.BranchLayoutFileReader.read(BranchLayoutFileReader.java:44)
	at com.virtuslab.gitmachete.frontend.file.highlighting.MacheteAnnotator.getParentBranchName(MacheteAnnotator.java:145)
	at com.virtuslab.gitmachete.frontend.file.highlighting.MacheteAnnotator.processMacheteGeneratedEntry(MacheteAnnotator.java:111)
	... 32 more
```

IDE error just because I've got some mess in machete file for a moment... we're gonna get flooded with IDE errors ☹️